src/components/LetterInputs.js

```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { update } from "../reducers/word";
+import { increment } from "../reducers/Increment";
+import { useSelector } from "react-redux";
+
+function LetterInputs({ wordLength }) {
+  const [letterStates, setLetterStates] = useState(Array(wordLength).fill(""));
+
+  const updateLetterState = (index, value) => {
+    const updatedStates = [...letterStates];
+    updatedStates[index] = value;
+    setLetterStates(updatedStates);
+  };
+
+  const clearInputFields = () => {
+    const clearedStates = Array(wordLength).fill("");
+    setLetterStates(clearedStates);
+  };
+
+  const dispatch = useDispatch();
+  const counterValue = useSelector((state) => state.increment.value);
+
+  return (
+    <>
+      <div style={{ display: "flex", alignItems: "center", gap: "1rem" }}>
+        {Array.from({ length: wordLength }, (_, i) => (
+          <input
+            maxLength={1}
+            key={i}
+            value={letterStates[i]}
+            onChange={(event) => updateLetterState(i, event.target.value)}
+          />
+        ))}
+      </div>
+      <button
+        className="submit-word-btn"
+        onClick={() => {
+          const guessedWord = letterStates.join("");
+          console.log(guessedWord);
+          dispatch(update(guessedWord));
+          clearInputFields();
```

Comment:
Do you want to leave a comment here that says something about changing the color  based on if it's right or not? Where would that go...

- when the word is guessed we'll need to check EACH individual one so...
- It seems prudent that our array of letters in this file may need a 'status' property to flip it's css on or off maybe? there would therefor be an additional check happening via a function in this file? 
- meanwhile, the full check that moves the next word can continue to happen, but perhaps the logic could be simplified to really just get a return value from this file (since the letters already will have been checked)? 
-